src/libraries/Microsoft.NETCore.Platforms/src/runtimeGroups.props

```diff
@@ -130,16 +130,6 @@
     </RuntimeGroup>
 
     <!-- linux mint includes major version in Include because Parent is different for each major -->
-    <RuntimeGroup Include="linuxmint.17">
-      <Parent>ubuntu.14.04</Parent>
-      <Architectures>x64</Architectures>
-      <Versions>1;2;3</Versions>
-    </RuntimeGroup>
-    <RuntimeGroup Include="linuxmint.18">
-      <Parent>ubuntu.16.04</Parent>
-      <Architectures>x64</Architectures>
-      <Versions>1;2;3</Versions>
-    </RuntimeGroup>
     <RuntimeGroup Include="linuxmint.19">
```

Comment:
Linux Mint 18 and previous are EOL April 2021 (https://endoflife.software/operating-systems/linux/linux-mint, https://linuxmint.com/download_all.php)